samwise.js

```diff
@@ -1,112 +1,103 @@
 /**
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!